jws.go

```diff
@@ -94,19 +95,38 @@ func (sig Signature) mergedHeaders() rawHeader {
 
 // Compute data to be signed
 func (obj JSONWebSignature) computeAuthData(payload []byte, signature *Signature) []byte {
-	var serializedProtected string
+	var buf bytes.Buffer
+
+	pro := new(rawHeader)
```

Comment:
This is a nit, but I'd prefer if we gave these variables more descriptive names. It's not clear at first glance what "pro", "enc" and "buf" mean or what they're used for.

Reply:
How about calling these protectedHeader, needsBase64, and authData (or something along those lines)?